### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by VishnuSanal** [VishnuSanal/Quotes#70] [Ideas]
# Brain dump for v3.0.0
### Feel free to voice your opinions!

**Comment by VishnuSanal** [comment]
Ref: #53

Current photos will no longer be supported (since we'll remove Firebase storage). We can persist the images user has already downloaded.
 - Question: Should we do this? Or just remove all those & start fresh?

So, on the new layout, there'll be a search bar at the top
 - Q: Maybe tags (chips) too?

Then, we can have our current lists -> one showing downloaded pics & the other fetching thumbnails of random pics with Unsplash API
 - Q: Or pics for specific keywords? Keyword suggestions?

Whilst searching, the pics changes according to the keyword used.

When a user clicks on a button, it'd download the image with download link, saves it & sets it as a background.

This is what I think about the user flow for new BG layout. Please give your suggestions too! :)

**Reply by shuvashish76** [reply]
> Should we do this? Or just remove all those & start fresh?

I think fresh start better.

> Maybe tags (chips) too?

I don't think it's important for now.

> Or pics for specific keywords? Keyword suggestions?

Instead of keyword suggestions we could use "Collection". Something like "[Staff Picks -studies](https://lichess.org/study/staff-picks)" feature of Lichess. Basically its just curated list of images picked by you (_best for quote background_). [Needs some brainstorming, _Low priority for now_]

> Whilst searching, the pics changes according to the keyword used.

I think "Live Preview" feature only takes internet data unnecessarily.

**Comment by shuvashish76** [comment]
![Screenshot_2022-02-11-23-03-38](https://user-images.githubusercontent.com/48226312/153641330-769462be-547f-48ba-9e3b-53a005dbb365.jpeg)

Currently #101 not possible as we've GMS & Firebase.
We're 1step closer with #53 (v3). @VishnuSanal any alternative for fonts?

**Reply by VishnuSanal** [reply]
> @VishnuSanal any alternative for fonts?

I'm thinking about the same too. All the fonts from Google fonts must be an overkill. I feel like we can use an OSS alternative to Firebase storage.
